api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -6020,8 +6021,8 @@
  				select distinct person.person_id, user.system_id, 1, CURRENT_TIMESTAMP, user.retired, user.retired_by, user.date_retired, user.retire_reason, CONCAT('prov', SUBSTRING(person.uuid, 5))
  				from person as person
  					inner join encounter as encounter on encounter.provider_id = person.person_id
- 					left join (select max(users.user_id) as user_id, users.system_id, users.person_id, users.retired, users.retired_by, users.date_retired, users.retire_reason 
- 							from users as users group by users.person_id) as user 
+ 					left join (select max(users.user_id) as user_id, users.system_id, users.person_id, users.retired, users.retired_by, users.date_retired, users.retire_reason
```

Comment:
@PawelGutkowski the problem with this is that it results into an attempt to insert duplicate records. The original  "group by users.person_id" ensured that this did not happen. But now after your replacing it with "group by users.system_id, users.person_id, users.retired, users.retired_by, users.date_retired, users.retire_reason", you will get duplicate key constraint violation errors. Because you did a fresh setup and hence did not have providers/encounters, you did not get any such errors. But try to do an upgrade to 2.0 that has a good number of records for such.